zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -1561,6 +1567,45 @@ mod test {
             .run();
     }
 
+    // maybe consider to move to mpt_circuit module
+    #[cfg(feature = "scroll")]
+    #[test]
+    fn call_non_exist_with_value_mpt_circuit() {
+        let callee_code = bytecode! {
+            .op_call(0xc350, 0xff, 0x13, 0x0, 0x0, 0x0, 0x0)
+        };
+
+        let ctx = TestContext::<2, 1>::new(
+            None,
+            account_0_code_account_1_no_code(callee_code),
+            tx_from_1_to_0,
+            |block, _tx| block.number(0xcafeu64),
+        )
+        .unwrap();
+
+        let block: GethData = ctx.into();
+        let mut builder = BlockData::new_from_geth_data_with_params(
+            block.clone(),
+            CircuitsParams {
+                max_rws: 1024,
+                max_copy_rows: 1024,
+                max_mpt_rows: 3500,
+                ..Default::default()
+            },
+        )
+        .new_circuit_input_builder();
+        builder
+            .handle_block(&block.eth_block, &block.geth_traces)
+            .unwrap();
+        let mut block = block_convert::<Fr>(&builder.block, &builder.code_db).unwrap();
+        block.mpt_updates.mock_fill_state_roots();
+        let mpt_circuit = MptCircuit::new_from_block(&block);
+        let prover = MockProver::<Fr>::run(12, &mpt_circuit, vec![]).unwrap();
+        println!("prover result: {:?}", prover.verify());
+        assert!(prover.verify().is_ok());
+        //prover.assert_satisfied_par();
```

Comment:
verify twice

Reply:
removed in [1a0e6d8](https://github.com/scroll-tech/zkevm-circuits/pull/949/commits/1a0e6d88d3e972ab4140fe590ad4e14a94780de8) @lispc